Updated fig4 results, new supp. material, increased focus on name origin analysis (#11)
greenelab/nature_news_manuscript: content/04.results.md

When we again compare _Nature_ articles against last authorship in _Nature_, we observe patterns consistent with the citation analysis with all predicted name origins, except for East Asian and Celtic/English closely matching the predicted name origin rate of last authors in _Nature_ (Figure {@fig:suppfig4}c, dark grey and purple lines).
We find a similar pattern when we consider _The Guardian_ articles, except we additionally find a depletion of predicted European name origins (Figure {@fig:suppfig4}c, dark grey and light grey lines).

To further understand the source of Celtic/English over-enrichment and East Asian under-enrichment, we selected a subset of quotes from people that were also cited in the news article; again, this analysis could only be completed on _Nature_ articles.
We found that the disparity was greatly reduced, more closely matching the analysis on citations alone (Figure {@fig:suppfig_quote_cite}a,b).
Next, we identified if the journalist name origin had any effect on quote disparities.
When considering all quotes, we found that journalists with a predicted East Asian name origin had a higher rate of East Asian quoted speakers (Table {@tbl:table5}).

> However, this disparity was reduced when only considering quotes from people cited within the same news article, and reduced further when considering only citations with a US affiliation (Table {@tbl:table6} and {@tbl:table7}).

Reviewer: I can't quite tell if the disparity that this refers to is the line 181 disparity or the line 178 disparity. Do we definitely mean disparity here?

Author: good point, I edited it to be "We found that the **_under-enrichment of East Asian name origins_**"

Author: changed in https://github.com/greenelab/nature_news_manuscript/pull/11/commits/86381c5ff0d5c571b051bbb12fd326d76303b8ba

When we again compare _Nature_ articles against last authorship in _Nature_, we observe patterns consistent with the citation analysis with all predicted name origins, except for East Asian and Celtic/English closely matching the predicted name origin rate of last authors in _Nature_ (Figure {@fig:suppfig4}c, dark grey and purple lines).
We find a similar pattern when we consider _The Guardian_ articles, except we additionally find a depletion of predicted European name origins (Figure {@fig:suppfig4}c, dark grey and light grey lines).

To further understand the source of Celtic/English over-enrichment and East Asian under-enrichment, we selected a subset of quotes from people that were also cited in the news article; again, this analysis could only be completed on _Nature_ articles.

> We found that the under-enrichment of East Asian name origins was greatly reduced, more closely matching the analysis on citations alone (Figure {@fig:suppfig_quote_cite}a,b).

Reviewer: ```suggestion
We found that the under-enrichment of predicted East Asian name origins was greatly reduced, more closely matching the analysis on citations alone (Figure {@fig:suppfig_quote_cite}a,b).
```

> Next, we designed an experiment to test if the predicted journalist name origin had any effect on quote disparities.

Reviewer: ```suggestion
Next, we designed an experiment to test if predicted journalist name origin had any effect on quote disparities.
```

We found that journalists with a predicted East Asian name origin had a higher rate of East Asian quoted speakers (26.0%) in comparison to journalists with Celtic/English (3.7%) or European (8.3%) predicted name origins (Table {@tbl:table5}).

> To interrogate the source of this difference between journalists with different predicted name origins, we looked at a more specific subsection of quotes.

Reviewer: ```suggestion
To examine the source of this difference between journalists with different predicted name origins, we looked at a more specific subsection of quotes.
```

> We added a new constraint that the quotes must be from a cited last author in the same news article (Table {@tbl:table6}), and that the article must have a US affiliation (Table {@tbl:table7}).

Reviewer: ```suggestion
We added a new constraint that the quotes must be from a cited last author in the same news article (Table {@tbl:table6}) and that the article must have a US affiliation (Table {@tbl:table7}).
```

> We found that differences between journalists with different predicted name origins was nearly eliminated when additionally considering quoted speakers with a US affiliated citation, as evidenced in the East Asian column of Table {@tbl:table7}.

Reviewer: ```suggestion
We found that differences between journalists with different predicted name origins was nearly eliminated when restricting to quoted speakers with a US affiliated citation, as evidenced in the predicted East Asian column of Table {@tbl:table7}.
```

We found that journalists with a predicted East Asian name origin had a higher rate of East Asian quoted speakers (26.0%) in comparison to journalists with Celtic/English (3.7%) or European (8.3%) predicted name origins (Table {@tbl:table5}).
To examine the source of this difference between journalists with different predicted name origins, we looked at a more specific subsection of quotes.
We added a new constraint that the quotes must be from a cited last author in the same news article (Table {@tbl:table6}) and that the article must have a US affiliation (Table {@tbl:table7}).
We found that differences between journalists with different predicted name origins was nearly eliminated when restricting to quoted and cited speakers, with marginal improvement with an additional restriction of US affiliated citations, as evidenced in the predicted East Asian column of Table {@tbl:table7}.
The differences between Table {@tbl:table5} and Tables {@tbl:table6} and {@tbl:table7}, indicate that the predicted name origin of a journalist has some effect on sources gathered outside of directly cited works.

> Since the additional location based constraint of having a US-affilitated citation induces only a marginal change, we are unable to identify if there is a significant regional bias.    

Reviewer: ```suggestion
The additional reduction in differences between the predicted name origin groups and quote rates for US-affiliated people could suggest that, while we don't know the locations of journalists at the time an article is written, regional differences in the presence of journalists may some role in driving the observed disparities.
```

> Furthermore, we do not know the location of the journalists at the time of writing the articles.

Reviewer: ```suggestion
```

> Nevertheless, our findings support the hypothesis that there exist name-origin specific biases of journalists when identifying sources beyond authors directly cited within an articles.

Reviewer: ```suggestion
```
